usecase/providerd.go

```diff
@@ -73,26 +73,36 @@ func (g *providerDaemon) Start(ctx context.Context) <-chan []error {
 
 		for {
 			select {
-			case <-ctx.Done():
```

Comment:
it doesn't make sense to remove context.Done() logic here. 
When user uses this daemon alone and they cannot control the daemon by using context....